controllers/authController.js

```diff
@@ -1,12 +1,33 @@
+const jwt = require("jsonwebtoken");
+
 const User = require("../models/User");
 const ERROR_MESSAGE = require("../constants");
 
 const postLogin = async (req, res, next) => {
-  const { email } = req.body;
+  const { accessToken } = req.cookies;
+  const { newAccessToken, newRefreshToken } = req;
+  const { ACCESS_SECRET_KEY } = process.env;
+  const { email } = accessToken
+    ? jwt.verify(accessToken, ACCESS_SECRET_KEY, { ignoreExpiration: true })
+    : jwt.verify(newAccessToken, ACCESS_SECRET_KEY);
```

Comment:
영진님 저희 같이 이야기한 것처럼 조건을 변경하면 ignoreExpiration 옵션을 따로 넣어주지 않아도 될 것 같아요~!

```suggestion
  const { email } = newAccessToken
    ? jwt.verify(newAccessToken, ACCESS_SECRET_KEY)
    : jwt.verify(accessToken, ACCESS_SECRET_KEY);
```

Reply:
넵 확인 했습니다-!